app/helpers/error_summary_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ErrorSummaryHelper
+  def anchor_link(record, field)
```

Comment:
I think `#field_anchor_link`, which is in the spec, is a better name

Reply:
I went back and forth on whether to put "field" in the module name or method name, and accidentally settled on the _worst option_, neither. Will re-introduce it.

Reply:
Addressed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/112/commits/a22c48dd0de9badb194a01e7f42c5a2c96fc338f